### DISCUSSION TITLE
for Drafts, Scheduled toots, lists:   show number of items inside

### DISCUSSION THREAD

**Discussion by stephan75** [tuskyapp/Tusky#2897] [Ideas]
When you have toots still as a draft, or saved for scheduled sending, but you are not sure whether there are any of them at all, you have to enter each "folder" to verify whether there are any items stored there.

Would it be possible to count those items and display that number in brackets after "Drafts", "Scheduled toots" and so on?

See manually modified screenshot (of an older Tusky version):
![image](https://user-images.githubusercontent.com/5250044/203162523-011b65ad-d3de-4994-ba6b-559e8d9861b9.png)

**Comment by charlag** [comment]
I can imagine motivation for drafts but what's the use case for lists and scheduled posts?
